Pillar_II/esm/README.md

```diff
@@ -348,27 +348,32 @@ may be useful for others with issues:
 
 #### Running from Alien4Cloud
 
-There is an environment initialization script:
-`/gpfs/projects/dese28/eflows4hpc/esm/fesom2/env.sh`
+Firstly, you must have a Alien4Cloud user account and have a vault ID (SSH key) and you have to add it to the `authorized_keys` file on the systems you want to run your workflows.
 
-This does the same initializations as in `launch_fesom2.sh`
-but also specifies cores, queue type and ensembles.
-
-Then in the A4C topology, in the `extra_compss_opts=`
-we specify the `--env_script`:
+##### How to get a vault ID
 
 ```bash
-$ compss... \
-        --qos=debug \
-        --exec_time=120 \
-        --keep_workingdir \
-        --worker_working_dir=/gpfs/projects/dese28/eflows4hpc/esm/fesom2/src \
-        --worker_in_master_cpus=48 \
-        --num_nodes=3 \
-        --pythonpath=/gpfs/projects/dese28/eflows4hpc/esm/fesom2/src:/apps/HECUBA/2.1_intel/compss \
-        --env_script=/gpfs/projects/dese28/eflows4hpc/esm/fesom2/env.sh \
-        --storage_props=/gpfs/projects/dese28/eflows4hpc/esm/fesom2/src/hecuba_lib/storage_props.cfg --storage_home=/apps/HECUBA/2.1_intel/compss
+docker run ghcr.io/eflows4hpc/hpcwaas-api:main-cli --api_url https://eflows4hpc.bsc.es/waas -u [your alien4cloud user]:***********= ssh_keys key-gen
 ```
+Running this will get you a SSH key ID which will be used in setting up TOSCA application. 
+
+##### Create your application 
+
+Login to your Alien4Cloud account and create a new application based on an existing topology template. Choose `fesom2 parameterised` as your topology template and proceed creating your application.
+
+##### Configuring topology and running your application
+
+Once you create your application. You will see an option to edit your `Environment`. 
+
+Proceed to edit your environment and navigate to `Prepare next deployment`. Here you will see options to modify your `Topology`, `Inputs`, `Location` etc.
+
+Feel free to edit your topology if needed. Since we created is based on `fesom2 parameterised` template so it has all the paths and parameters like --start_date --hpc etc pre-configured that you can see in the `arguments` field. The main script that is configured to be launched in this template is `/gpfs/projects/dese28/eflows4hpc/workflow-registry/Pillar_II/esm/src/launch_fesom2.sh` which you can see in the `command` field in your topology. 
```

Comment:
s/is based/it based ?

s/template so it has/template, it has/?

s/like --start_date --hpc etc/like `--start_date` `--hpc` etc.

And add some breaks to the line (other paragraphs can be used as example).